lightning/src/util/mod.rs

```diff
@@ -15,10 +15,14 @@ pub(crate) mod fuzz_wrappers;
 pub mod events;
 pub mod errors;
 pub mod ser;
+pub mod message_signing;
 
 pub(crate) mod byte_utils;
 pub(crate) mod chacha20;
+#[cfg(feature = "fuzztarget")]
+pub mod zbase32;
 #[cfg(not(feature = "fuzztarget"))]
+pub(crate) mod zbase32;
```

Comment:
The compile error in CI is that `poly1305` is no longer `#[cfg(not(feature = "fuzztarget"))]` (ie not compiled in fuzzing). You need to duplicate the cfg line, above.